### PR TITLE
Docker compose bugfix

### PR DESCRIPTION
**Pull request by oscardimanno**
## Description
Corrected the docker-compose installation
Not working php modules installation and starting mysql service.

## Testing
Just runned docker compose build to check

## Changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code is written according to the guidelines found [here] (https://github.com/php-fig/fig-standards).
- [x] I have signed the Contributor Licence Agreement between me and YetiForce.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,7 +25,24 @@ RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources
 RUN apt-get update
 RUN apt-get install -y --no-install-recommends mariadb-server mariadb-client
 RUN apt-get install -y --no-install-recommends nginx nginx-extras zip unzip cron nodejs npm yarn mc htop openssh-server git
-RUN apt-get install -y --no-install-recommends "php${PHP_VER}"-{fpm,mysql,curl,intl,gd,bcmath,soap,ldap,imap,xml,cli,zip,json,opcache,mbstring,apcu,imagick}
+# RUN apt-get install -y --no-install-recommends "php${PHP_VER}"-{fpm,mysql,curl,intl,gd,bcmath,soap,ldap,imap,xml,cli,zip,json,opcache,mbstring,apcu,imagick}
+RUN apt-get install -y --no-install-recommends "php${PHP_VER}"-fpm
```
[the remark this answers is not in the document]
Comment (oscardimanno) [reply Dockerfile]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (oscardimanno) [reply Dockerfile]: @sonatype-lift ignore

File: Dockerfile [hunk 2]
```diff
@@ -43,7 +60,7 @@ COPY ./tests/setup/docker_entrypoint.sh /
 RUN rm /var/www/html/.user.ini
 RUN rm /var/www/html/public_html/.user.ini
 
-RUN	service mysql start; \
+RUN	service mariadb start; \
```
[the remark this answers is not in the document]
Comment (oscardimanno) [reply Dockerfile:46]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (oscardimanno) [reply Dockerfile:46]: @sonatype-lift ignore

File: Dockerfile (hunk 1 above)
Comment (mariuszkrzaczkowski) [inline Dockerfile]: what does it change?

Comment (oscardimanno) [reply Dockerfile]: apt doesn't picking regexp for some stranger things..and just edited to install one to one module. I could also install all with a single call to apt like: apt install php-fpm php-mysql etc..

Comment (mariuszkrzaczkowski) [reply Dockerfile]: this is a better option
 apt install php-fpm php-mysql etc..

Comment (oscardimanno) [reply Dockerfile]: sure. I will correct and commit this in some hours

**Review by oscardimanno — COMMENTED** [review]
@mariuszkrzaczkowski check it now!

File: Dockerfile [hunk 3]
```diff
@@ -25,7 +25,7 @@ RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources
 RUN apt-get update
 RUN apt-get install -y --no-install-recommends mariadb-server mariadb-client
 RUN apt-get install -y --no-install-recommends nginx nginx-extras zip unzip cron nodejs npm yarn mc htop openssh-server git
-RUN apt-get install -y --no-install-recommends "php${PHP_VER}"-{fpm,mysql,curl,intl,gd,bcmath,soap,ldap,imap,xml,cli,zip,json,opcache,mbstring,apcu,imagick}
+RUN apt-get install -y --no-install-recommends php${PHP_VER}-fpm php${PHP_VER}-mysql php${PHP_VER}-curl php${PHP_VER}-intl php${PHP_VER}-gd php${PHP_VER}-bcmath php${PHP_VER}-soap php${PHP_VER}-ldap php${PHP_VER}-imap php${PHP_VER}-xml php${PHP_VER}-cli php${PHP_VER}-zip php${PHP_VER}-json php${PHP_VER}-opcache php${PHP_VER}-mbstring php${PHP_VER}-apcu php${PHP_VER}-imagick
```
[the remark this answers is not in the document]
Comment (oscardimanno) [reply Dockerfile:28]: @sonatype-lift ignore

**Pull request merged**